app/src/pages/inside/dashboardItemPage/modals/common/widgetControls/cumulativeTrendControls.jsx

```diff
@@ -113,10 +113,20 @@ export class CumulativeTrendControls extends Component {
         },
       },
     });
+
+    this.levelOneAttributes = [];
```

Comment:
Instead of duplicating entered values you can get them from the form using [formValueSelector](https://redux-form.com/7.3.0/docs/api/formvalueselector.md/)